src/diffusers/models/attention_processor.py

```diff
@@ -684,6 +685,49 @@ def __call__(self, attn: "Attention", hidden_states, encoder_hidden_states=None,
         return hidden_states
 
 
+class CrossFrameAttnProcessor:
```

Comment:
Very cool to reuse the `AttnProcessor` logic here, could you maybe add it to the pipeline file instead though as it's quite a specific processor?

Reply:
Done!